### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by what-the-hub**
add sidebar markup

### CODE REVIEW

File: sass/style.sass [hunk 1]
```diff
@@ -225,6 +256,51 @@ label
...
+      opacity: 70%
+
+.menu
+  background: $white
+  height: 100vh
+  position: fixed
+  top: 0
+  z-index: 5
+  width: 300px
+  outline: none
```
Comment (silchencko) [inline sass/style.sass]: What is it for here?

File: index.html [hunk 2]
```diff
@@ -8,274 +8,304 @@
...
-            <div class="item">
-                <img alt="Bezos"
-                     src="https://kod.ru/content/images/2020/08/jeff-bezos-amazon-el-hombre-mas-rico-del-mundo_1520365805_1.jpg_1834093470.jpg">
-                <div class="item-container">
-                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus consequuntur corporis dolor,
-                        dolorem ducimus eos, error eum excepturi facilis illo itaque laboriosam neque quae quia
-                        repudiandae.
-                        Deleniti mollitia praesentium voluptatibus.</p>
+    <div class="sticky-header">
+        <i class="fa fa-bars"></i>
```
Comment (silchencko) [inline index.html:12]: `cursor: pointer`

**Review by silchencko — APPROVED** [review]
Looks good to me! Just have some minor comments. But it's good enough to go!

**Comment by what-the-hub** [comment]
> Looks good to me! Just have some minor comments. But it's good enough to go!
@sikchencko
Thanks, got it! I will review this

**Pull request merged**